source/Calamari.Shared/Deployment/Conventions/StageScriptDependenciesConvention.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System.IO;
+using Calamari.Common.Commands;
+using Calamari.Common.Features.Packages;
+using Calamari.Common.Plumbing.FileSystem;
+using Calamari.Common.Plumbing.Logging;
+using Calamari.Common.Plumbing.Variables;
+using Calamari.Deployment.Conventions.DependencyVariablesStrategies;
+
+namespace Calamari.Deployment.Conventions
+{
+    public class StageScriptDependenciesConvention : IInstallConvention
```

Comment:
This is essentially a copy of `StageScriptPackagesConvention`, but with support for different variables.

Reply:
`StageScriptPackagesConvention` will eventually be removed once the feature toggle is gone.

Reply:
Maybe let's go `StageDependenciesConvention`